src/activity-functions-function-notation.xml

```diff
@@ -753,19 +807,79 @@
 
         <p>Your success in this course is almost entirely dependent on your understanding of this simple concept: <em>The <q>blank</q> method.</em></p>
 
-        <p>When you see a formula like <m>f(x) = x^2 - x</m>. It means this:<me>f(\text{blank}) = (\text{blank}) - (\text{blank})</me></p>
+        <p>When you see a formula like <m>f(x) = x^2 - x</m>. It means this:<me>f(\text{blank}) = (\text{blank})^2 - (\text{blank})</me></p>
 
         <p>At least that's the way formulas really work.</p>
 
-        <p>Using this function as an example, <m>f(5)</m> means <q>The formula with a <m>5</m>in it</q>. Literally it means place number <m>5</m> in all the blanks. As in<me>f(5) = (5)^2 - (5) = 20</me></p>
+        <p>Using this function as an example, <m>f(5)</m> means <q>The formula with a <m>5</m> in it</q>. Literally it means place number <m>5</m> in all the blanks.</p>
+
+        <exercise>
+            <webwork>
+                <setup>
+                    <var name="$five">
+                        <static>5</static>
+                    </var>
+                    <var name="$out">
+                        <static>20</static>
+                    </var>
+                    <var name="$M">
+                        <static>M</static>
+                    </var>
+                    <var name="$Mexp">
+                        <static>M^2-M</static>
+                    </var>
+                    <var name="$Q">
+                        <static>Q</static>
+                    </var>
+                    <var name="$Qexp">
+                        <static>Q^2-Q</static>
+                    </var>
+                    <var name="$P">
+                        <static>x-12</static>
+                    </var>
+                    <var name="$expression">
+                        <static>(x-12)^2-(x-12)</static>
+                    </var>
+                    <pg-code>
+                        $five = Real(5);
+                        $out = Real(20);
+                        Context()->variables->are(M=>'Real', Q=>'Real', x=>'Real');
+                        $M=Compute("M");
+                        $Q=Compute("Q");
+                        $P=Compute("x-12");
+                        $Mexp=Compute("M**2-M");
+                        $Qexp=Compute("Q**2-Q");
+                        $expression=Compute("(x-12)**2-(x-12)");
+                    </pg-code>
+                </setup>
+
+                <statement>
```

Comment:
Also, you can use `\text{blank}`, or you can use the `<fillin width="5" />` element, which creates a fill-in-the-blank underline. You can't literally fill it in; it is not the same as a WeBWorK answer blank. The "5" means it will roughly be 5 characters long.